iib/workers/tasks/build_merge_index_image.py

```diff
@@ -247,17 +274,22 @@ def handle_merge_request(
         ]
 
         if deprecation_bundles:
+            intermediate_image_name = _get_external_arch_pull_spec(
```

Comment:
this function returns string only (pull spec - URL) it will not be changed unless we use different parameters.
If you are referring to actual image and that we are checking `is_image_fbc` before build and push then I would say it will not be changed - the image will be still FBC or SQLite only content will be different.
And this `is_image_fbc` will be removed once we merge the deprecate - so I can move this check down a bit in that PR. 

I put it here at top because at  that time I thought it is better to fail before time consuming build and push.

Reply:
DIscussed in a meeting.. I understand now